.travis/docker.mk

```diff
@@ -7,7 +7,7 @@ travis_start: ## Start containers stopped by `travis_stop`
 	docker-compose -f .travis/docker-compose-travis.yml start
 
 travis_test: ## Run tests on Docker containers, as on Travis CI
-	docker exec -it edx_notes_api env TERM=$(TERM) /edx/app/edx_notes_api/edx_notes_api/.travis/run_tests.sh
+	$(TOX)pytest
```

Comment:
Can you set `TOXENV` (like `TERM` above) and update `run_tests.sh` to respect it instead?  That would still let you run the tests in the Docker container, and presumably avoid the need to switch to the `test_es_disabled` settings.